public/dnd.php

```diff
@@ -0,0 +1,33 @@
+<html>
+    <head>
+        <title>DND stuff</title>
+    </head>
+    <body>
+        <?php
+        /**
+         * Created by PhpStorm.
+         * User: Luka_offline
+         * Date: 15/04/2018
+         * Time: 20:13
+         */
+
+        $dice_data = $_GET["roll"];
+
+        $dice_arr = preg_split("/d|\+/", $dice_data);
```

Comment:
Most of this looks good, except for this bit here.

Plus signs are really hit and miss in URLs, they are often interpreted as spaces.

What if we split out the GET variables instead?
* n => number of dice
* d => sides per dice
* o => offset

Reply:
I need to check if nightbot can do that, for now, my next commit assumes + works well :P